src/target/arc_regs.h

```diff
@@ -61,6 +61,15 @@
 #define AUX_DC_CTRL_REG			0X48
 #define DC_CTRL_IM			(1 << 6)
 
+/* L2 cache registers */
+#define AUX_L2_IVL2_REG 		0x905
+#define L2_IVL2_INVALIDATE		(1)
+#define AUX_L2_FLL2_REG 		0x904
```

Comment:
It is better to sort registers in AUX number order and to name defines like an actual register they represent, e.g. `SLC_AUX_CACHE_FLUSH` in this case. Since AUX is in the name of register, I think it is better to preserve naming from PRM, then to force AUX into the beginning of the define.